src/libstd/path/posix.rs

```diff
@@ -59,6 +60,12 @@ pub fn is_sep(c: char) -> bool {
     c == SEP
 }
 
+impl fmt::Show for Path {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "Path {{ {} }}", self.display())
```

Comment:
Since it's `Show`, to differentiate it from just a string property. Similar to how this PR's `Rc` is `Rc {{ {:?} }}` instead of just showing the internal value.